### DISCUSSION TITLE
The ability to Customize Objects in Taiko Circles for the Argon Skin

### DISCUSSION THREAD

**Discussion by Knifs** [ppy/osu#22891] [Ideas]
I would honestly love to be able to customize the Objects in Taiko Circles in the Argon Skin with something like a Star or a smiley face. As much as I love the Arrows, they're kind of distracting for me so I decided to make some experimental ones to kind of see the result.
![image (5)](https://user-images.githubusercontent.com/40337532/226071102-edea8785-3606-42a3-a737-eeb6dabe223f.png)

**Comment by respektive** [comment]
I would really like to have this feature. I really like the argon skin, but I wanted to edit the hitcircles to be a fully opaque circle. I tried to drop a taikohitcircle.png file in the skin editor but that didn't actually replace it. I had the idea to instead make a separate skin with just the few taiko elements i wanna change and import that into lazer, however that always defaults back to the classic skin for other missing assets.

So as a potential solution, I think it could be nice to have an option to change the fallback skin to any of the other default skins, including argon.
